lib/runner/cli/clirunner.js

```diff
@@ -348,13 +348,23 @@ CliRunner.prototype = {
       }
     }
 
-    if (envs.length > 1) {
+    if (envs.length > 1 && (this.settings.testWorkers == null || !this.settings.testWorkers.enabled)) {
```

Comment:
the property name should follow the existing style of declaring properties in nightwatch.json - as in `test_workers`.